services/common/rs_server_common/utils/utils.py

```diff
@@ -107,18 +107,21 @@ def validate_str_list(parameter: str, handler: ValidatorFunctionWrapHandler) ->
 
 
 def validate_inputs_format(
-    interval: str,
+    date_time: str,
     raise_errors: bool = True,
-) -> Tuple[Union[None, datetime], Union[None, datetime]]:
+) -> Any:
     """
     Validate the format of the input time interval.
 
     This function checks whether the input interval has a valid format (start_date/stop_date) and
     whether the start and stop dates are in a valid ISO 8601 format.
 
     Args:
-        interval (str): The time interval to be validated, with the following format:
+        date_time (str): The time value to be validated, with the following formats:
             "2024-01-01T00:00:00Z/2024-01-02T23:59:59Z"
+            "../2024-01-02T23:59:59Z"
+            "2024-01-01T00:00:00Z/.."
+            "2024-01-01T00:00:00Z"
```

Comment:
@Padeanu about the rest of this docstring: 
```python
    Returns:
        Tuple[Union[None, datetime], Union[None, datetime]]:
            A tuple containing:
            - start_date (datetime): The start date of the interval.
            - stop_date (datetime): The stop date of the interval.
        Or [None, None] if the provided interval is empty.

    Note:
        - The input interval should be in the format "start_date/stop_date"
        (e.g., "2022-01-01T00:00:00Z/2022-01-02T00:00:00Z").
        - If there is an error, err_code and err_text provide information about the issue.
```
Is the `Returns` still up-to-date ? I think you know return 3 values in the tuple. And I think the Note is not up-to-date, maybe you can remove it since the input format is already explained in the `Args` above.

Reply:
Updated